src/ios/ListLibraryItems.m

```diff
@@ -167,8 +167,8 @@ - (void)uploadItem:(CDVInvokedUrlCommand *)command {
 
                             [request setValue:[[[NSFileManager defaultManager] sizeOfItemAtURL:mLocalTempURL] stringValue] forHTTPHeaderField:@"Content-Length"];
                             [request setValue:[[NSFileManager defaultManager] md5OfItemAtURL:mLocalTempURL] forHTTPHeaderField:@"Content-MD5"];
-                            NSURLSessionUploadTask * task = [session uploadTaskWithRequest:request fromFile:mLocalTempURL];
-                            [task resume];
+                            currentTask = [session uploadTaskWithRequest:request fromFile:mLocalTempURL];
+                            [currentTask resume];
```

Comment:
I don't think these changes should be in this PR?

Reply:
You're right, all changes concerning this deserves to be in the timeout PR.